pydoctor/templatewriter/__init__.py

```diff
@@ -84,7 +94,7 @@ def __init__(self, name: str, text: str):
         self.text = text
         """Template text: contents of the template file."""
 
-    TEMPLATE_FILES_SUFFIX = ('.html', '.css', '.js')
+    TEMPLATE_FILES_SUFFIX = ('.html', '.xml', '.xhtml', '.css', '.js', '.svg')
```

Comment:
I'm curious: why are you adding XML and XHTML?

For XML, it seems strange to treat that as HTML later, since most XML formats are not HTML serializations.

For XHTML, the Twisted templating system only accepts XHTML as far as I know, so we could consider renaming the template files from `.html` to `.xhtml`.

Reply:
Yeah, I can roll back this changes, it was because all-document template was xhtml at the beginning but it could not get rendered by the browser with this file extension so everything uses .html. 

I'll roll back this